### ISSUE TITLE
using oscar 19

### ISSUE THREAD

**Issue by kiwinesian** [open-osp/open-osp#98]
Hi there,

just looking to get an instruction on how to install Oscar 19.

I have put either of the following conditions in the `local.env` but doesn't seem to pull the development oscar 19 branch.

```
OSCAR_TREEISH=oscar19.1
OSCAR_REPO=git@bitbucket.org:oscaremr/oscar.git
```

```
OSCAR_TREEISH="oscar19.1"
OSCAR_REPO="git@bitbucket.org:oscaremr/oscar.git"
```

during the `./openosp bootstrap` it still switched to the `release/Oscar-BC-15` branch.

Steps that I performed:

Clone the git
./openosp setup and fill in all information
edit the `local.env` to include the line above
./openosp setup
./openosp bootstrap

the 2nd `./openosp setup` almost doesn't really do anything.

looking forward to the guidance here! :)

**Comment by countable** [comment]
Hmm, you shouldn't need to run openosp setup a 2nd time. I've updated the docs to reflect this. We haven't tested Oscar 19 in quite a while. Let me know if you have any issues!

**Comment by kiwinesian** [comment]
> Hmm, you shouldn't need to run openosp setup a 2nd time. I've updated the docs to reflect this. We haven't tested Oscar 19 in quite a while. Let me know if you have any issues!

hi there,

thanks for getting back very soon!

just destroyed the docker, `rm` the mariadb volume and rebuilt it
Still stuck on `+ branch=release/Oscar-BC-15` while executing `./openops bootstrap`

I do notice that it should load `local.env`.
However, seems like it's not reading the `OSCAR_TREEISH` variable during build?

**Comment by ferdinandpolpol** [comment]
Fixed this in #112, closing.

**Comment by countable** [comment]
Cool, so it works on Oscar 19 now?
